packages/ui/src/views/labels/components/label-tag.tsx

```diff
@@ -16,7 +16,7 @@ export const LabelTag: FC<LabelTagProps> = ({ scope, withIndicator = false, wrap
       size="sm"
       icon={scope !== undefined ? scopeTypeToIconMap[getScopeType(scope)] : undefined}
       variant="secondary"
-      className="grid grid-flow-col"
+      className="grid grid-flow-col content-center"
       labelClassName="grid grid-flow-col content-center"
       valueClassName="grid grid-flow-col content-center"
```

Comment:
Do we really these classes? Without these classes we are achieving the layout already. Can you please tell the exact purpose of these classes?

<img width="719" height="805" alt="Image" src="https://github.com/user-attachments/assets/fa810adc-b13e-4583-a3d7-40534e093ae8" />

Reply:
Yes, we absolutely need them. We discussed this when we had a call last week. After the call I tried the solution with max-width: 50%, but it's not working properly. We should keep it the way it presented.
cc @praneshg239